met-web/src/apiManager/endpoints/index.ts

```diff
@@ -0,0 +1,9 @@
+import { AppConfig } from '../../config';
```

Comment:
Curious , is there a better way of importing than absolute imports?
Like Vue had @ which helps to use something like 
import AccountChangeMixin from '@/components/auth/mixins/AccountChangeMixin.vue'
I am not sure if react we can achieve this.but putting out this here so that you can check

Reply:
You mean letting go of the relative path with the '../../' ? I reviewed it on https://medium.com/geekculture/making-life-easier-with-absolute-imports-react-in-javascript-and-typescript-bbdab8a8a3a1. It seems doable and let's add a new ticket for it